en/lessons/sustainable-authorship-in-plain-text-using-pandoc-and-markdown.md

```diff
@@ -495,17 +496,37 @@ LaTeX is installed, a beautifully formatted PDF file can be created
 using the same command structure:
 
 ```
-$ pandoc -o main.pdf main.md
+$ pandoc main.md -o main.pdf
+```
+
+If your document is written in languages other than English, you will likely need to use
+the XeLaTeX engine instead of plain LaTeX for .pdf conversion:
+
+```
+pandoc main.md --pdf-engine=xelatex -o main.pdf
 ```
 
-With time, you will be able to fine tune the formatting of PDF documents
-by specifying a LaTeX style file (saved to the same directory), and
-running something like:
+Make sure your tex editor supports the UTF-8 encoding. When using XeLaTeX for conversion into
+.pdf, instead of the `fontfamily` attribute in YAML, specify the `mainfont` attribute, to
+produce something like the following:
 
 ```
-$ pandoc -H format.sty -o project.pdf --number-sections --toc project.tex
+    ---
+    title: Plain Text Workflow
+    author: Dennis Tenen, Grant Wythoff
+    date: January 20, 2014
+    mainfont: times
+    ---
 ```
 
+Note that YAML often replicates some, although not all, of the command line argument (flag)
```

Comment:
This is a useful feature of Pandoc to note. However, please convert to a footnote since it is a bit of an in-the-weeds sidebar to the main flow of the tutorial - great to have available, but not in a way that disrupts the flow of the lesson.

Reply:
Will do.